backend/inspirehep/search/facets.py

```diff
@@ -271,7 +281,7 @@ def flattened_refereed = [];
                         "aggs": {
                             "citation_buckets": {
                                 "range": {
-                                    "field": "citation_count",
+                                    "field": field,
```

Comment:
do we really want to change the name here? we're not changing it for the stats computed from it. If you want to communicate the info that self-citations are excluded, maybe add a meta field `excludes-self-citations: true/false`?

Reply:
We have to as citation_count without self citations is stored in this field, and we want to generate bucket aggregation for it isn't it? 
Why we want to add this meta gield? When user requests for citation count without self-citations he know what he requested already isn't he?